.gitignore

```diff
@@ -11,3 +11,6 @@ dist/
 
 # Test artifacts
 internal/e2e/testdata/**/moves.tf
+
+# Envrc
+.envrc
```

Comment:
I agree that this should be added to .gitignore 👍

However I think we can make the comment clearer:
```suggestion
# Local direnv configuration
.envrc
```